setup-deploy-keys

```diff
@@ -49,6 +49,7 @@ deploy_to cockpit-project/cockpit-weblate \
 # cockpit-machines
 deploy_to cockpit-project/cockpit-machines \
     --deploy-from \
+        cockpit-project/cockpit-machines/npm-update/SELF_DEPLOY_KEY \
```

Comment:
The commit message for "setup-deploy-keys: add cockpit-machines to node-cache" says "we rename the COCKPIT_DEPLOY_KEY to SELF_DEPLOY_KEY", but this commit didn't actually do that.